english/flight/flight.md

```diff
@@ -1,4 +1,4 @@
 The chococopter sees you and lowers down near your position. As you get on and look
-around you see a gas mask and a fireman suit.
+around you see a gas mask and a fireman suit.  Upon further inspection you find a laser.  Your only capable of carrying 2 items.

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.